models/topic.go

```diff
@@ -54,11 +54,39 @@ func (err ErrTopicNotExist) Error() string {
 	return fmt.Sprintf("topic is not exist [name: %s]", err.Name)
 }
 
-// ValidateTopic checks topics by length and match pattern rules
+// ValidateTopic checks a topic by length and match pattern rules
 func ValidateTopic(topic string) bool {
 	return len(topic) <= 35 && topicPattern.MatchString(topic)
 }
 
+// SanitizeAndValidateTopics sanitizes and checks an array or topics
+func SanitizeAndValidateTopics(topics []string) (validTopics []string, invalidTopics []string) {
+	validTopics = make([]string, 0)
+	invalidTopics = make([]string, 0)
+
+	LOOP_TOPICS:
```

Comment:
To not have use go-to labels first create map with cleaned up topics and then loop thorough keys to validate them

Reply:
Comment still stands ;)

Reply:
I felt it was bad code but hit my limits of go knowledge 😄. Did you mean something like this?